### ISSUE TITLE
Problem with pagination when using a bottom header for Total count

### ISSUE THREAD

**Issue by ljmoleiro** [xaksis/vue-good-table#745]
### Issue Type (delete the irrelevant ones) 

- [x] Bug

### Specs
"version": "2.21.0",

What browser?
chrome Version 84.0.4147.125 (Official Build) (64-bit)

### Expected Behavior

I've made a bottom header for Totals Count, like the one you explain in https://github.com/xaksis/vue-good-table/issues/115#issuecomment-384039249.

The data displays correctly but I got an issue with pagination

### Actual Behavior
The pagination take in count this bottom header row, so if I get 5 rows of data by default, its actually displays four and takes in count the row for Totals counts. So I get four rows of data and the five is the bottom header

### Steps to Reproduce the Problem
Please detail your steps here

1. this is the template

```
 <vue-good-table
              :columns="columns"
              :rows="rows"
              fixed-header
              :isLoading.sync="isLoading"
              :search-options="searchOptions"
              :pagination-options="paginationOptions"
              :totalRows="totalRecords"
              @on-page-change="onPageChange"
              @on-per-page-change="onPerPageChange"
              :group-options="groupOptions"
      >
        <template slot="table-header-row" slot-scope="props">
            <span v-if="props.column.field === 'created_at'">
             Totals
            </span>
        </template>
      </vue-good-table>
```
1.
this is my script code
 ```
  isLoading: false,
        rows: [],
        columns: [
          {
            label: 'Created At',
            field: 'created_at',
            type: 'date', dateInputFormat: 'yyyy-MM-dd H:m:s',
            dateOutputFormat: 'dd-MM-yyyy',
          },
          {
            label: 'Token Amount',
            field: 'token_amount',
            type: 'number',
            headerField: this.sumTokenAmount,
          },
          {
            label: 'Price Per Token ',
            field: 'cost_per_token',
            type: 'number',
            headerField: this.sumCostPerToken,
          },
          {
            label: 'Total Price',
            field: 'total_price',
            type: 'number',
            headerField: this.sumTotalPrice,
          },           
          {
            label: 'User',
            field: 'user_id',
          },       
        ],
        serverParams: {
          params: {
            columnFilters: {},
            sort: {
              field: '',
              type: '',
            },
            page: 1,
            perPage: 5,        
          }
        },
        totalRecords: 0,
        searchOptions: {
          enabled: true,
          placeholder: 'Search this table',
        },
        paginationOptions: {
          enabled: true,
          mode: 'pages',
          perPage: 5,
          position: 'bottom',
          perPageDropdown: [5, 10, 15],
          dropdownAllowAll: false,
          setCurrentPage: 1,
          nextLabel: 'next',
          prevLabel: 'prev',
          rowsPerPageLabel: 'Rows per page',
          ofLabel: 'of',
          pageLabel: 'page', // for 'pages' mode
          allLabel: 'All',
        },
        groupOptions: {
          enabled: true,
          headerPosition: 'bottom',
        }

      }
```
1. I fetch de data from backend and store de data as in the example 
  ```
 fetchItems(config = null) {
        axios.get('/api/info/get_info', config).then(({data}) => {
          this.rows = [
            {
              name: 'Totals',
              children: data.data
            }];
          this.totalRecords = data.total
          Object.assign(this.paginationOptions, {
            setCurrentPage: data.currentPage
          })
        })
      },
```

Hope you can bring some light on this. Thanks

**Comment by xaksis** [comment]
Should be fixed in 2.21.6

**Comment by abrah-am** [comment]
Hi @xaksis I see that you closed this issue, however, I see the same behavior in 2.21.6. I also upgraded to version "vue-good-table": "2.21.10", but still see the problem. Any ideas to get this fixed or are you planning a fix?
